.appveyor.yml

```diff
@@ -0,0 +1,160 @@
+
+  # Make sure the Python version is in PATH, and others are not.
+  - ps: |
+      # Remove the wrong Python version(s) from PATH.
+      $p = $env:PATH -split ";" | Where-Object {
+             -not (Test-Path -Path "$_\python.exe") -and
+             -not (Test-Path -Path "$_\pip.exe")
+           }
+      # Add binary dir for `pip --user` packages.
+      $p += "$env:APPDATA\Python\Scripts"
+      # Add python27-x64.
+      $p += "c:\Python27-x64"
+      $p += "c:\Python27-x64\Scripts"
+      $env:PATH = $p -join ";"
+
+  # Pip on Appveyor is too old. Install a recent version in our user dir.
+  - python -m pip install --upgrade --user pip
+
+  # Install Python packages.
+  - pip install --upgrade --user pywin32 yapf
+
+  # Add Rust/Cargo to PATH.
+  - ps: $env:PATH += ";$env:CARGO_HOME\bin"
+
+  # Install Rust via rustup-init, unless it was restored from cache.
+  - ps: |
+      if (-not (Test-Path -Path $env:CARGO_HOME)) {
+        Invoke-WebRequest -Uri "https://win.rustup.rs" `
+                          -OutFile "$env:TEMP\rustup-init.exe"
+        Exec { & "$env:TEMP\rustup-init.exe" -y }
+      }
+
+  # Update Rust.
+  - rustup update
+
+  # The Rustup directory is very big with many files, which makes saving and
+  # restoring the Appveyor cache noticably slow. Remove unnecessary stuff.
+  # TODO: Use `rustup component remove docs` instead, when this issue
+  # is resolved: https://github.com/rust-lang-nursery/rustup.rs/issues/998.
+  - ps: |
+      Remove-Item -Recurse -Force -ErrorAction Ignore -Path @(
+        "$env:RUSTUP_HOME\downloads",
+        "$env:RUSTUP_HOME\tmp",
+        "$env:RUSTUP_HOME\toolchains\stable-x86_64-pc-windows-msvc\share\doc"
+      )
+      $null  # If Remove-Item encountered an error, don't stop the build.
+
+  # Log installed Node.js version + processor architecture.
+  - node -p "`Node ${process.version} ${process.arch}`"
+
+  # Log installed Python version + processor architecture.
+  - ps: >-
+      @("from sys import version",
+        "print 'Python', version") -join "`n" | & python -
+
+  # Log some more versions.
+  - pip --version
+  - rustc --version
+  - cargo --version
+
+  # Log environment variables.
+  - ps: |
+      Get-ChildItem env:* | Select-Object -Property Name, @{
+        Name = "Value"
+        Expression = {
+          if ($_.Name -like "*PATH") { $_.Value -replace ";", ";`n" }
+          else { $_.Value }
+        }
+      } | Format-Table -AutoSize -Wrap
+
+before_build:
+  # Appveyor may have restored actual files/dirs, which should be symlinks.
+  # This can happen if the cache is saved after a failed build.
+  - ps: Remove-Cache-Symlinks
+
+  # Download clang and gn, generate ninja files.
+  - python tools\setup.py
+
+  # Synchronize mtime_cache. This makes it possible for build products to be
+  # considered up-to-date between builds.
+  # TODO: Use ccache or something similar.
+  - ruby tools/mtime_cache.rb **/*.{%{cpp}} -c .mtime_cache/cache.json
```

Comment:
Does that glob match .h and .cc files?

Reply:
Yes, and I'm not using mtime_cache any more, at least for now.